Source/Private/ARTUtils.h

```diff
@@ -0,0 +1,24 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface ARTUtils : NSObject
+
+#pragma mark - Random generator
+
++ (void)seedRandomWithNumber:(unsigned int)number;
++ (void)seedRandomWithSystemTime;
++ (double)randomFraction;
++ (long)randomFromZeroToNumber:(long)number;
+
+#pragma mark - Incremental backoff and jitter // RTB1
+
++ (double)jitterDelta;
++ (double)generateJitterCoefficient;
++ (double)backoffCoefficientForRetryNumber:(NSInteger)retryNumber;
```

Comment:
Does this need to be part of the public interface of this class?

Reply:
https://github.com/ably/ably-cocoa/pull/1520/commits/b17d6b572f236a2f2147f83324ad0fb336f32ec1